yb-voyager/cmd/common.go

```diff
@@ -405,67 +402,37 @@ func CreateMigrationProjectIfNotExists(dbType string, exportDir string) {
 			utils.ErrExit("couldn't create sub-directories under %q: %v", filepath.Join(projectDirPath, "schema"), err)
 		}
 	}
-	migUUID, err := generateAndStoreMigrationUUIDIfRequired(exportDir)
-	if err != nil {
-		utils.ErrExit("couldn't generate/store migration UUID: %w", err)
-	}
 
-	err = metadb.CreateAndInitMetaDBIfRequired(exportDir)
+	createInitConnectToMetaDBIfRequired()
+	setSourceDbType(dbType)
+}
+
+func createInitConnectToMetaDBIfRequired() {
```

Comment:
Just name it as `initMetaDB()` -- no need to list all the implementation details in the name.